.github/workflows/gh-ci-cron.yaml

```diff
@@ -0,0 +1,203 @@
+      with:
+        python-version: 3.9
+        auto-update-conda: true
+        channel-priority: flexible
+        channels: conda-forge, bioconda
+        mamba-version: "*"
+        add-pip-as-python-dependency: true
+        architecture: x64
+
+    - name: install_dev_versions
+      run: |
+        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scipy
+
+    - name: install_other_deps
+      uses: ./.github/actions/setup-deps
+      with:
+        mamba: true
+        numpy: ""
+        scipy: ""
+
+    - name: build_srcs
+      uses: ./.github/actions/build-src
+      with:
+        build-hole: true
+        build-tests: true
+        build-docs: false
+
+    - name: run_tests
+      run: |
+        pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+
+
+  # Issue #3442
+  native_march:
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: setup_os
+      uses: ./.github/actions/setup-os
+      with:
+        os-type: "ubuntu"
+
+    - name: setup_miniconda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: 3.9
+        auto-update-conda: true
+        channel-priority: flexible
+        channels: conda-forge, bioconda
+        mamba-version: "*"
+        add-pip-as-python-dependency: true
+        architecture: x64
+
+    - name: install_deps
+      uses: ./.github/actions/setup-deps
+
+    - name: set_extra_flags
+      run: |
+        sed -i "s/#extra_cflags =/extra_cflags = -march=native -mtune=native/g" package/setup.cfg
+        cat package/setup.cfg
+
+    - name: build_srcs
+      uses: ./.github/actions/build-src
+      with:
+        build-hole: true
+        build-tests: true
+        build-docs: false
+
+    - name: run_tests
+      run: |
+        pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+
+
+  # Issue 1727
+  pip-only:
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: setup_os
+      uses: ./.github/actions/setup-os
+      with:
+        os-type: "ubuntu"
+
+    - name: setup_miniconda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: 3.9
+        auto-update-conda: true
+        add-pip-as-python-dependency: true
+        architecture: x64
+
+    - name: install_minimum_pip
+      run: |
+        pip install numpy Cython pytest-xdist
+
+    - name: build_srcs
+      uses: ./.github/actions/build-src
+      with:
+        build-hole: false
+        build-tests: true
+        build-docs: false
+
+    - name: run_tests
+      run: |
+        pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+
+
+  # Issue 3216
+  old-chemfiles:
```

Comment:
I'm not sure we should be maintaining support for older chemfiles versions.  The package is pre 1.0 and will continually be unstable, so we're signing up for a lot of work.

Reply:
Currently our code paths enable support for both pre and post chemfiles 0.10. I agree that we probably shouldn't waste time trying to support old pre-1.0 here, in which case we should just drop support for it (in a separate PR).

I've raised a separate PR. Let's keep this in for now and remove it in the follow-up PR?